dagmc_stats.dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM  ubuntu:18.04
+
+# set timezone
+ENV HOME /root
+ENV TZ=America/Chicago
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# ubuntu setup
+RUN apt-get update
+RUN apt-get install -y --fix-missing \
+    software-properties-common \
+    wget \
+    build-essential \
+    python-setuptools \
+    python-numpy \
+    python-pip \
+    python-tables \
+    python-matplotlib \
+    git \
+    cmake \
+    vim \
+    emacs \
+    nano \
+    gfortran \
+    autoconf \
+    libtool \
+    cpio \
+    libblas-dev \
+    liblapack-dev \
+    libhdf5-dev
+
+RUN pip install Cython pytest
+
+# build MOAB
+RUN cd $HOME \
+    && mkdir opt \
+    && cd opt \
+    && mkdir moab \
+    && cd moab \
+    && git clone https://bitbucket.org/fathomteam/moab \
+    && cd moab \
+    && git checkout -b Version5.1.0 origin/Version5.1.0 \
+    && autoreconf -fi \
+    && cd .. \
+    && mkdir build \
+    && cd build \
+    && ../moab/configure --enable-shared --enable-optimize --enable-pymoab --disable-debug --with-hdf5=/usr/lib/x86_64-linux-gnu/hdf5/serial --prefix=$HOME/opt/moab \
+    && make \
+    && make install \
+    && cd .. \
+    && rm -rf build moab
+
+# set environment variables
+ENV PATH $HOME/opt/moab/bin/:$PATH
+ENV LD_LIBRARY_PATH $HOME/opt/moab/lib:$LD_LIBRARY_PATH
+ENV PYTHONPATH $HOME/opt/moab/lib/python2.7/site-packages/:$PYTHONPATH
+ENV PYTHONPATH /root/project/scripts:$PYTHONPATH
```

Comment:
I strongly advice against this, I would suggest either:
- install your python package in the system and rely on them to run the test
- run test from project folder and do import in test such as `from scripts import dagmc_stats`

Reply:
Right- So then do I need to setup a `setup.py` script then to install in the system? Like @piperlincoln did for DAGMC-viz?

This was the only way I could get the module to be found when in the testing directory without  doing a system install.

Reply:
no you don't need to install, I don't think @piperlincoln setup does not work properly anyway and she did not rely on installed packages to run the test.

Reply:
Do you have a good example of how do this properly?

Reply:
@piperlincoln does it correctly in her PR just ignore the setup.py :)

basically with your layout you need:
- run test from `.../project/`
- in the `test.py` `import scripts.dagmc_stats` or `from scripts import dogma_stats`

Reply:
If I run the tests from `../project` which is the default behavior on CI, it can't find any of the test geometry .h5m files. Those are located in the test folder. Also- I want the test files to be able to run on a local machine. I don't think this method allows that.

Reply:
I don't understand why `pytest` can't load your package when not running in a docker container. (I tried can't figure this out...)
otherwise best practice advice to do a setup.py
https://docs.pytest.org/en/latest/goodpractices.html

Reply:
If I run pytest in the main dir (not the test dir) it can't find the geometry files. I just added a setup.py (need to push still) so it can find the module just fine now wherever I am. But it still can't find the geometry files in the test dir to actually run the tests.

Reply:
now you should be able to run from the test folder :)